Sources/XiEditor/Info.plist

```diff
@@ -53,7 +67,5 @@
 	<string>Main</string>
 	<key>NSPrincipalClass</key>
 	<string>NSApplication</string>
-	<key>NSSupportsAutomaticGraphicsSwitching</key>
```

Comment:
Not sure if you want to remove this, AFAIK this makes xi-mac use the discrete GPU whenever possible and that eats a bunch of unnecessary battery.

Reply:
Xcode removed it as duplication. There's another `NSSupportsAutomaticGraphicsSwitching` in the file.

Reply:
Gotcha. Seems like an all around good change then.